vpdq/python/test_vpdq_hash.py

```diff
@@ -1,48 +1,67 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 import vpdq
-import typing as t
 import pytest
 import test_util
 import os
-import pathlib
+from pathlib import Path
 
-SAMPLE_HASH_FOLDER = "/vpdq/sample-hashes"
-SAMPLE_VIDEOS = "/tmk/sample-videos"
+SAMPLE_HASH_FOLDER = Path("vpdq/sample-hashes")
+SAMPLE_VIDEOS = Path("tmk/sample-videos")
 FFMPEG = "/usr/bin/ffmpeg"  # please customize according to your installation (but do not commit)
 SECOND_PER_HASH = 1
 DISTANCE_TOLERANCE = 10
 QUALITY_TOLERANCE = 50
+TEST_FILES = [
+    "chair-19-sd-bar",
+    "chair-20-sd-bar",
+    "chair-22-sd-grey-bar",
+    "chair-22-sd-sepia-bar",
+    "chair-22-with-large-logo-bar",
+    "chair-22-with-small-logo-bar",
+    "chair-orig-22-fhd-no-bar",
+    "chair-orig-22-hd-no-bar",
+    "chair-orig-22-sd-bar",
+    "doorknob-hd-no-bar",
+    "pattern-hd-no-bar",
+    "pattern-longer-no-bar",
+    "pattern-sd-grey-bar",
+    "pattern-sd-with-large-logo-bar",
+    "pattern-sd-with-small-logo-bar",
+]
 
 
 class TestVPDQHash:
-    def test_VPDQ_hash(self):
-        d = os.path.dirname(os.path.dirname(os.getcwd()))
-        hash_folder = pathlib.Path(d + SAMPLE_HASH_FOLDER)
-        video_folder = pathlib.Path(d + SAMPLE_VIDEOS)
+    def test_compare_hashes(self):
+        """This regression test is creating hashes from sample videos and compare them with the provided hashes line by line.
+        Two VPDQ features are considered the same if each line of the hashes are within DISTANCE_TOLERANCE.
+        For hashes that have a quality lower than QUALITY_TOLERANCE, the test will skip them for comoparing.
+        """
+        d = Path.cwd().parent.parent
+        hash_folder = d / SAMPLE_HASH_FOLDER
+        video_folder = d / SAMPLE_VIDEOS
         self.test_hashes = {}
         self.sample_hashes = {}
```

Comment:
nit: Why make these instance members?
```suggestion
        test_hashes = {}
        sample_hashes = {}
```